metals/src/main/scala/scala/meta/metals/Main.scala

```diff
@@ -15,20 +15,69 @@ import scala.meta.internal.metals.clients.language.MetalsLanguageClient
 import scala.meta.internal.metals.logging.MetalsLogger
 
 import org.eclipse.lsp4j.jsonrpc.Launcher
+import coursierapi.Complete
+import scala.jdk.CollectionConverters._
+import scala.util.Try
 
 object Main {
   def main(args: Array[String]): Unit = {
     if (args.exists(Set("-v", "--version", "-version"))) {
+      val api: Complete = coursierapi.Complete
+        .create()
+        .withInput("org.scalameta:mtags")
+
+      val supported =
+        BuildInfo.supportedScalaVersions.toSet
+
+      import scala.collection.parallel._
+
+      val additional =
+        api
+          .complete()
+          .getCompletions()
+          .asScala
+          .toVector
+          .flatMap { case artif =>
+            Try {
+              val scalaVer = artif.stripPrefix("mtags_")
+              val List(epoch, major, rest) = scalaVer.split("\\.", 3).toList
+
+              if (epoch == "2" && major.toInt < 11) None // 2.10 and below
+              else if (rest.endsWith("-NIGHTLY")) None // dotty nightlies
+              else if (supported(scalaVer))
+                None // versions we already know about
+              else if (epoch == "0") None // old dotty versions
+              else if (scalaVer.contains("-M")) None // milestone versions
+              else Some(scalaVer)
+            }.toOption.flatten
+          }
+          .toParArray
```

Comment:
```suggestion
          .distinct
          .toParArray
```
We can have duplicates here, maybe it's worth doing `distinct` before next api calls